apps/addons/management/commands/approve_addons.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+from django.core.management.base import BaseCommand, CommandError
+
+import amo
+from addons.models import Addon
+from editors.helpers import ReviewHelper
+
+
+class Command(BaseCommand):
+    args = '<addon_guid addon_guid ...>'
+    help = 'Approve a list of addons, given their GUIDs.'
+
+    def handle(self, *args, **options):
+        if len(args) == 0:  # No GUID provided?
+            raise CommandError(
+                'Please provide at least one addon guid to approve.')
+
+        addons = Addon.with_unlisted.filter(
+            guid__in=args,
+            status__in=[amo.STATUS_NOMINATED, amo.STATUS_UNREVIEWED,
+                        amo.STATUS_PENDING])
+
+        for addon in addons:
+            print addon
+
+        confirm = raw_input(
+            'Are you sure you want to bulk approve and sign all those {0} '
+            'add-ons? (yes/no)'.format(len(addons)))
+        if confirm != 'yes':
+            raise CommandError("Aborted.")
+
+        for addon in addons:
+            helper = ReviewHelper(request=None, addon=addon,
+                                  version=addon.latest_version)
+            # Provide the file to review/sign to the helper.
+            helper.set_data({'addon_files': [addon.latest_version.files],
+                             'comments': 'bulk approval'})
+            if addon.status in [amo.STATUS_NOMINATED, amo.STATUS_PENDING]:
```

Comment:
Ok, I tracked this down, it should be:

`if addon.status in [amo.STATUS_NOMINATED, amo.STATUS_LITE_AND_NOMINATED, amo.STATUS_PUBLIC]:`

`STATUS_PUBLIC` is for pending updates.